cardano-cli/src/Cardano/CLI/Shelley/Orphans.hs

```diff
@@ -10,41 +10,42 @@
 {-# LANGUAGE UndecidableInstances #-}
 
 {-# OPTIONS_GHC -Wno-orphans #-}
+{-# LANGUAGE LambdaCase #-}
 
 module Cardano.CLI.Shelley.Orphans () where
 
-import           Cardano.Prelude
-
-import           Control.SetAlgebra as SetAlgebra
-import           Data.Aeson
-import qualified Data.ByteString.Base16 as Base16
-import qualified Data.ByteString.Short as SBS
-import qualified Data.Text.Encoding as Text
-
 import           Cardano.Api.Orphans ()
-
+import           Cardano.Ledger.AuxiliaryData (AuxiliaryDataHash (..))
```

Comment:
Can you undo the import changes to follow the style in other modules?

Reply:
Any chance we could use `ImportQualifiedPost` language extension like we do in `plutus-apps` so I could just do a line sort to keep my imports tidy?

Reply:
We would need to make a decision to use `ImportQualifiedPost` in the repo. Having a mix of some modules using it and others not is not ideal. I personally don't see justification for this aesthetic change.